src/frontend/EdFi.DataManagementService.Frontend.AspNetCore/EdFi.DataManagementService.Frontend.AspNetCore.nuspec

```diff
@@ -1,17 +1,18 @@
 <?xml version="1.0"?>
 <package >
   <metadata>
-    <id>EdFi.DataManagementService.Frontend.AspNetCore</id>
+    <id>EdFi.DataManagementService</id>
```

Comment:
Some day we may need to distinguish multiple front-end applications from each other. Or, we might just have a single package with multiple available front-end projects. We may need to change the name in the future, but for right now this short name is good enough. Thank you for changing it.